learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -0,0 +1,348 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import '../../components/complexity.dart';
+import '../../components/page_container.dart';
+import '../../config/theme/colors_provider.dart';
+import '../../constants/assets.dart';
+import '../../constants/colors.dart';
+import '../../constants/sizes.dart';
+
+class WelcomeScreen extends StatelessWidget {
+  const WelcomeScreen();
+
+  @override
+  Widget build(BuildContext context) {
+    return PageContainer(
+      child: SingleChildScrollView(
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: const [
+            Expanded(child: _SdkSelection()),
+            Expanded(child: _TourSummary()),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _SdkSelection extends StatelessWidget {
+  const _SdkSelection();
+
+  @override
+  Widget build(BuildContext context) {
+    return _SdkSelectionBody(
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.fromLTRB(50, 60, 50, 20),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: const [
+                _IntroText(),
+                SizedBox(height: TobSizes.size32),
+                _Buttons(),
+              ],
+            ),
+          ),
+          Image.asset(TobAssets.welcomeLaptop),
```

Comment:
May we consider removing this image?

Reply:
Why do you want it removed? This image is in the approved layout:
https://www.figma.com/file/gzeYx965q4DwV6ugDzp0iG/Apache-Beam-Playground?node-id=1475%3A5359

There is a problem with it however. It is cropped nicely on the mock, but looks awkward when scrolled down:

![image](https://user-images.githubusercontent.com/44893228/186637874-edbb51d7-3896-4e38-aa97-804f1d28bfc3.png)

I suggest we keep it but stick to the bottom at the crop ratio from the mock so the user cannot scroll below this:

![image](https://user-images.githubusercontent.com/44893228/186638431-2d7b26f9-e26e-4837-ade3-0b225888ea0f.png)

Also for some reason the border shows hard pixels, we should see how to get them smooth if we keep the image.

Reply:
The image is displayed properly if the html renderer is selected. I'll update README.

Reply:
Fixed the layout, keeping the cropped laptop.